### DISCUSSION TITLE
Love this tracker! Best one yet. But need a way to reset when it loses your face.

### DISCUSSION THREAD

**Discussion by gweddlenz** [Shubhamai/StableView#68] [Ideas]
When I turn too much or otherwise go off camera I have to stop and start to get it to work again. If there is no way to recapture one's face, could we at least have some sort of keyboard shortcut to stop and start it?

**Comment by Shubhamai** [comment]
Hi, currently working on the next update to fix this issue. [here's a beta version](https://github.com/Shubhamai/StableView/releases/tag/v0.0.3-beta). Would love to get your feedbacks before the release.

Feel free to let me know if there are any other issues/improvements I should take a look at. 

Have a nice day!

**Comment by gweddlenz** [comment]
Thank you!

You have no idea how happy I am to hear this. I use StableView every day, it beats everything else out there imo (even payware). I will definitely be giving this a go and provide feedback!

Cheers,
Gary

***@***.***

**Comment by gweddlenz** [comment]
So I have been running this new version for a few days and it's great! It always recaptures my head. I have no problems with it now. Thanks very much!
